examples/gitlab-centos7-atomicapp/README.md

```diff
@@ -0,0 +1,157 @@
+postgresql-ytsy5   1/1       Running   0          15m
+redis-s5cge        1/1       Running   0          15m
+
+$ kubectl logs gitlab-0nax4
+which: no docker in
+(/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
+which: no docker in
+(/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin)
+Generating SSH2 RSA host key: [  OK  ]
+Generating SSH2 ECDSA host key: [  OK  ]
+Generating SSH2 ED25519 host key: [  OK  ]
+Waiting for database server to accept connections
+Setting up GitLab for firstrun. Please be patient, this could take a while...
+PG::Error: ERROR:  new encoding (UTF8) is incompatible with the encoding of the
+template database (SQL_ASCII)
+HINT:  Use the same encoding as in the template database, or use template0 as
+template.
+: CREATE DATABASE "gitlab_production" ENCODING = 'unicode'
+[..]
+Couldn't create database for {"adapter"=>"postgresql", "encoding"=>"unicode",
+"database"=>"gitlab_production", "host"=>"10.254.213.29", "port"=>5432,
+"username"=>"gitlab", "password"=>"password", "pool"=>10}
+Migrating database...
+Compiling assets. Please be patient, this could take a while...
+Starting supervisord...
+2015-09-08 12:09:17,509 CRIT Supervisor running as root (no user in config
+file)
+2015-09-08 12:09:17,509 WARN Included extra file "/etc/supervisord.d/cron.ini"
+during parsing
+2015-09-08 12:09:17,509 WARN Included extra file "/etc/supervisord.d/nginx.ini"
+during parsing
+2015-09-08 12:09:17,509 WARN Included extra file
+"/etc/supervisord.d/sidekiq.ini" during parsing
+2015-09-08 12:09:17,509 WARN Included extra file "/etc/supervisord.d/sshd.ini"
+during parsing
+2015-09-08 12:09:17,509 WARN Included extra file
+"/etc/supervisord.d/unicorn.ini" during parsing
+2015-09-08 12:09:17,569 INFO RPC interface 'supervisor' initialized
+2015-09-08 12:09:17,569 CRIT Server 'unix_http_server' running without any HTTP
+authentication checking
+2015-09-08 12:09:17,576 INFO supervisord started with pid 1
+2015-09-08 12:09:18,581 INFO spawned: 'sidekiq' with pid 307
+2015-09-08 12:09:18,584 INFO spawned: 'unicorn' with pid 308
+2015-09-08 12:09:18,590 INFO spawned: 'cron' with pid 309
+2015-09-08 12:09:18,594 INFO spawned: 'nginx' with pid 310
+2015-09-08 12:09:18,597 INFO spawned: 'sshd' with pid 311
+2015-09-08 12:09:19,993 INFO success: sidekiq entered RUNNING state, process
+has stayed up for > than 1 seconds (startsecs)
+2015-09-08 12:09:19,993 INFO success: unicorn entered RUNNING state, process
+has stayed up for > than 1 seconds (startsecs)
+2015-09-08 12:09:19,993 INFO success: cron entered RUNNING state, process has
+stayed up for > than 1 seconds (startsecs)
+2015-09-08 12:09:19,993 INFO success: nginx entered RUNNING state, process has
+stayed up for > than 1 seconds (startsecs)
+2015-09-08 12:09:19,993 INFO success: sshd entered RUNNING state, process has
+stayed up for > than 1 seconds (startsecs)
+
+```
+
+To forward port `30000` of ADB box to host machine port `30000`, add following line your Vagrantfile
```

Comment:
This should be automated by the ADB guys, could you please open an issue with them?